README.md

```diff
@@ -103,26 +98,30 @@ Make sure the config.json file doesn't have uuid, registered_at or deviceId popu
 
 Then run the supervisor like this:
 ```bash
-./tools/dev/dindctl run --image username/resin-supervisor:master --preload
+./dindctl run --image resin/amd64-supervisor:master --preload
 ```
-This will make the docker-in-docker instance pull the image specified in apps.json before running the supervisor.
+This will make the docker-in-docker instance pull the image specified in apps.json before running the supervisor, simulating a preloaded ResinOS image (where `resin preload` has been used on the image, see [the resin CLI docs](https://docs.resin.io/tools/cli/#preload-60-image-62-)).
```

Comment:
`apps.json` should be an inline code block, this is the convention we use for file names and commands.